### ISSUE TITLE
Not working with current Glimpse version?

### ISSUE THREAD

**Issue by nmehlei** [aaronpowell/glimpse-knockout#3]
I was unable to get this plugin to run with the current Glimpse version (1.4.2). Do I need to register the tabs somehow to be able to see them? Glimpse.axd does not list the knockout tab, even though the NuGet package is installed and the library is referenced.

What am I doing wrong?

**Comment by aaronpowell** [comment]
I believe there are some issues with extensions with the new HUD, but I haven’t had time to test it with the new HUD yet myself.

Aaron Powell
MVP - Internet Explorer (Development) | FunnelWeb Team Memberhttp://funnelweblog.com/

http://apowell.mehttp://apowell.me/ | http://twitter.com/slace | Skype: aaron.l.powell | Githubhttp://github.com/aaronpowell/ | BitBuckethttp://hg.apwll.me/

**Comment by johnny-chan** [comment]
Yes, i've been trying to use this extension with the glimpse (1.4.2) . It keeps throwing up a " 'jQueryGlimpse' is undefined " error message. I'm assuming this is one of the issues causing this. 

I am able to view the knockout tab in Glimpse, but i'm unable to see the underlying observable values within my viewModel. Each property in my view returns a value like this...

function observable() {if (arguments.length > 0) {// Write// Ignore writes if the value hasn't changedif ((!observable['equalityComparer']) || !observable['equalityComparer'](_latestValue, arguments[0])) { observable.valueWillMutate(); _latestValue = arguments[0];if (DEBUG) observable._latestValue = _latestValue; observable.valueHasMutated();}return this; // Permits chained assignments}else {// Read ko.dependencyDetection.registerDependency(observable); // The caller only needs to be notified of changes if they did a "read" operationreturn _latestValue;}} 

regards,

johnny

**Comment by rolf-compete-nl** [comment]
Hi

I am not sure if the solution I have implemented is valid but for now I have wrapped the Knockout-glimpse.js in a document ready function. It seems to work as expected now.

ciao
Rolf

**Comment by overflew** [comment]
Cheers Rolf!

**Comment by robcube** [comment]
Thanks Rolf for the "timeless" advice.  I just added as the first line of the glimpse-knockout.js file:

``` javascript
$(document).ready(function() {
```

then added a new line at the bottom. 

``` javascript
}); //document.ready
```

That's it!

Using Glimpse 1.8.6/MVC 5/knockout-3.2.0.js.

**Comment by dyardyGIT** [comment]
This didn't work with the latest release for me.  I got the latest from git, rebuilt and ensured the dlls' from git were used instead of those delivered with nuget.  The nuget package is out of date.
